background/lookup.js

```diff
@@ -239,7 +252,7 @@ export default class Lookup {
         ? 'emailHash'
         : method;
 
-      return fetch(`https://id.parity.io:8443/?${lookupMethod}=${input}`)
+      return fetch(`${this.lookupURL}/?${lookupMethod}=${input}`.replace(/\/+/g, '/'))
```

Comment:
It's not realy future proof (imagine if `input` can be something more then email in future). I think it would be better to trim possible trailing `/` from `lookupURL` instead. Also it should be done when altering the options not with every request.